Library/Formula/corectl.rb

```diff
@@ -0,0 +1,39 @@
+require "language/go"
+
+class Corectl < Formula
+  desc "CoreOS over OS X made very simple"
+  homepage "https://github.com/TheNewNormal/corectl"
+  url "https://github.com/TheNewNormal/corectl/archive/v0.0.18.tar.gz"
+  sha256 "8646c18ec0414a8f6e6330bc6720d9f0c4356ae7168b18cbc371220fe7a3e087"
+  head "https://github.com/TheNewNormal/corectl.git", :branch => "golang"
+
+  depends_on "go" => :build
+  depends_on "godep" => :build
+  depends_on "gnu-sed" => :build
+  depends_on :macos => :yosemite
+
+  def install
+    ENV["PATH"] = ENV["PATH"].sub("#{Superenv.bin}:", "")
```

Comment:
We'll need to fix that issue in that case then, we can't just remove this from the PATH.